libappstream-glib/as-app-validate.c

```diff
@@ -709,7 +709,8 @@ as_app_validate_icons (AsApp *app, AsAppValidateHelper *helper)
 		break;
 	case AS_ICON_KIND_LOCAL:
 		icon_name = as_icon_get_filename (icon);
-		if (!g_str_has_prefix (icon_name, "/")) {
+		if (icon_name == NULL &&
```

Comment:
Oops, I was confused.

I want to check if icon_name is `NULL` because `NULL` is also invalid so `==` is right, but shouldn't be `&&` with `g_str_has_prefix`.

Thank your for reminding me.